pkg/csi-common/utils.go

```diff
@@ -95,7 +97,7 @@ func RunControllerandNodePublishServer(endpoint string, d *CSIDriver, cs csi.Con
 
 func logGRPC(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
 	log.Debugf("GRPC call: %s", info.FullMethod)
-	logRedactedRequest(req)
+	log.Debugf("GRPC request: %+v", pbSanitizer.StripSecretsCSI03(req).String())
```

Comment:
`StripSecretsCSI03 ` doesn't work for CSI spec v1.0+  - https://github.com/kubernetes-csi/csi-lib-utils/blob/master/protosanitizer/protosanitizer.go#L49-L54. This driver uses `v1.1.0`, so we should switch to using `StripSecrets`

Reply:
As `pbSanitizer.StripSecrets(req).String()` is a string, we should change the format specifier to `%s` instead of `%+v`